packages/mobile/src/screens/chat-screen/ReactionPopup.tsx

```diff
@@ -151,15 +158,27 @@ export const ReactionPopup = ({
     [userId, handleClosePopup, dispatch, chatId, userIdEncoded]
   )
 
-  const handleCopyPress = useCallback(
-    (message: string) => {
-      Clipboard.setString(message)
-      handleClosePopup()
+  const { message: messageText } = message
```

Comment:
super nit and up to you, but I could almost prefer doing `message.message` rather than destructuring so we don't have `message`, `messages`, _and_ `messageText`

Reply:
I would tend to agree but `message.message` grinds on me more than the multiple variables 😆 
I wish we had named the outer object and the inner field different things (`message.text` or `chat.message`)

Reply:
yeeeah. could have called it chatMessage.message but that's not much better :joy: oh well, there's two hard problems in programming - cache invalidation, naming things, and off by one errors